app/models/ability.rb

```diff
@@ -6,14 +6,20 @@ def initialize(user) # rubocop:disable all
       case user.view_mode
       when 'superuser'
         can :manage, :all
+        can :note, Reservation
```

Comment:
We don't need to add explicit permissions to superusers and admins since the `can :manage, :all` basically gives them unlimited permissions (you'll note that we then explicitly **deny** permissions from admins as opposed to adding anything else).